wagtail/admin/tests/pages/test_explorer_view.py

```diff
@@ -403,9 +403,9 @@ def test_breadcrumb_uses_specific_titles(self):
 
         expected = (
             """
-            <li class="w-h-full w-flex w-items-center w-overflow-hidden w-transition w-duration-300 w-whitespace-nowrap w-flex-shrink-0 w-font-bold w-max-w-0" data-breadcrumb-item hidden>
+            <li class="w-h-full w-flex w-items-center w-overflow-hidden w-transition w-duration-300 w-whitespace-nowrap w-flex-shrink-0 w-font-bold" data-breadcrumb-item hidden>
                 <a class="w-flex w-items-center w-h-full w-text-primary w-px-0.5 w-text-14 w-no-underline w-outline-offset-inside hover:w-underline hover:w-text-primary w-h-full" href="%s">
-                                    Secret plans (simple page)
```

Comment:
try to avoid whitespace changes where possible - especially in tests that should not change due to our added code.